### DISCUSSION TITLE
clang-tidy support for extended floating-point types

### DISCUSSION THREAD

**Discussion by blackteahamburger** [microsoft/vscode-cpptools#10595] [Q&A] [answered]
With

    "C_Cpp.default.cppStandard": "c++23",
    "C_Cpp.default.compilerPath": "/usr/bin/g++-13"

The following code:

    int main()
    {
        _Float16 f;
    }

causes `_Float16 is not supported on this target C/C++(clang-diagnostic-error)`.

However, running `clang-tidy` directly produces no warnings or errors.

**Comment by sean-mcmanus** [comment, marked as answer]
@blackteahamburger Thanks for reporting this. I've moved it to https://github.com/microsoft/vscode-cpptools/issues/10610 so the bug fix can be tracked.

**Reply by sean-mcmanus** [reply]
The fix is available with 1.15.0 (pre-release): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.15.0
